src/main/java/teammates/common/util/CustomLogic.java

```diff
@@ -0,0 +1,26 @@
+package teammates.common.util;
+
+/**
+ * Stores various custom logic that are used across classes.
+ */
+public final class CustomLogic {
+
+    private CustomLogic() {
+        // utility class
+    }
+
+    /**
+     * Custom equals method that is able to handle null objects.
+     * @param first first object to be compared
+     * @param second second object to be compared
+     * @return true of both objects are null or equal, false otherwise.
+     */
+    public static boolean customEquals(Object first, Object second) {
```

Comment:
I think you should put the method into `EntityAttributes` since it is about comparison between `attributes`. In addition, please don't call it as `customEquals`. The name is not constructive as how "custom" is "custom". A name such as `equalsWithNullComparaision` (of course it is not a ideal name) is far better than `customEquals.`

Reply:
Moved custom equals method to `EntityAttributes`, and renamed it to reflect its purpose more clearly